bitcoin/src/psbt/map/input.rs

```diff
@@ -70,11 +70,11 @@ const PSBT_IN_PROPRIETARY: u8 = 0xFC;
 #[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
 pub struct Input {
     /// The non-witness transaction this input spends from. Should only be
-    /// [std::option::Option::Some] for inputs which spend non-segwit outputs or
+    /// `Option::Some` for inputs which spend non-segwit outputs or
     /// if it is unknown whether an input spends a segwit output.
     pub non_witness_utxo: Option<Transaction>,
     /// The transaction output this input spends from. Should only be
-    /// [std::option::Option::Some] for inputs which spend segwit outputs,
+    /// `Option::Som`e for inputs which spend segwit outputs,
```

Comment:
typo

Reply:
Thanks man! Appreciate you taking the time to review. Fixed.